src/hooks/transactions/useResetTransactionState.ts

```diff
@@ -19,15 +19,15 @@ import { useUpdateTransactionContext } from '../../contexts/TransactionContext';
 import { TransactionActionCreators } from '../../actions/transactionActions';
 import usePrevious from '../react/usePrevious';
 
-const useResetTransactionState = (type: string | number | undefined) => {
+const useResetTransactionState = (action: string | number | undefined) => {
```

Comment:
I would rename this to something like `incAction` in order to avoid confusion with state `action`

Reply:
IMHO i think that it should remain as `action`, because this parameter called `action` is the `action` from the state and i think it makes sense to have the mental link about it.